packages/emoji-blast-site/src/components/code-tag/code-tag.tsx

```diff
@@ -0,0 +1,10 @@
+import * as usageStyles from "../usage-container/styles";
```

Comment:
Aside: in theory I'd normally suggest that styles not be shared across components like this. It can get hard to detangle/refactor styles en masse. But, this pattern is already done, and I'm redoing the site in #405 -> #406 anyway.

Reply:
Noted